ui/k-toolbar/main.kytos

```diff
@@ -41,6 +54,161 @@
                 }
                 this.$kytos.$emit("showInfoPanel", listWindows)
             },
-        }
+            /*
+                Gets the data fields after button click and consumes POST api to post maintenace window.
+            */
+            post_window(){
+                var self = this
+                this.formatInfo();
+                $.ajax({
+                    async: true,
+                    dataType: "json",
+                    type: "POST",
+                    contentType: "application/json",
+                    data: JSON.stringify({"id": self.id,
+                                          "description": self.description,
+                                          "start": self.start,
+                                          "end": self.end,
+                                          "items": self.merged_list,         
+                                          "status": self.status,
+                    }),
+                url:this.$kytos_server_api + "kytos/maintenance",
+                success: function(data) {
```

Comment:
@rohanCodes001 both the `success` and `error` callbacks here work great, however our team is inclined to use the `done` and `fail` request callback as you've done for the `loadNames` method, could you refactor this part accordingly?